src/wp-includes/rest-api/endpoints/class-wp-rest-widgets-controller.php

```diff
@@ -439,6 +484,21 @@ protected function permissions_check( $request ) {
 		return true;
 	}
 
+	/**
+	 * Look for "lost" widgets.
```

Comment:
I'd add `@see retrieve_widgets` to explain what "lost" means

Reply:
Fixed